prog-o-meter.py

```diff
@@ -277,6 +283,26 @@ def read_days_file(_filename):
     days_text.close() 
     return days
 
+def get_completion_date(days_remaining):
+    """Calculate the date at which the challenge will be over.
+
+    Args:
+        days_remaining: number of days remaining in the project
+
+    Returns:
+        The project completion date as a string
+    """
+    
+    today = datetime.date.today()
+    completion_date = today + datetime.timedelta(days=days_remaining)
+
+    if 4 <= completion_date.day <= 20 or 24 <= completion_date.day <= 30:

```

Comment:
I know this is not standard in all code, but for our project, I would like all control flows to be documented. Try and take a look at some of the other if/else statements in the code, and add similar style comments to your line 299 and line 301

Reply:
Got it, I had a look around your code and I see what you mean. I'll do that now.

Reply:
Do you want me to comment the if and else separately? or shall I just comment the if to explain what the whole thing does?

Reply:
Great, thanks :) I just want to keep the code easy to read for new coders